uber/site_sections/registration.py

```diff
@@ -167,6 +167,18 @@ def history(self, session, id):
                                .order_by(Tracking.when).all()
         }
 
+    @ajax
+    def resend_email(self, session, id):
+        message = {''}
+        if id:
+            try:
+                email = session.query(Email).filter(Email.model == 'Attendee', Email.fk_id == id).all()[0]
```

Comment:
Was there a particular reason you didn't use the built-in SQLAlchemy method `first()` instead of accessing the first element of the all() dict?